waltz-test/src/main/java/com/wepay/waltz/test/util/IntegrationTestHelper.java

```diff
@@ -46,6 +46,9 @@
 public class IntegrationTestHelper {
 
     private static final int STORAGE_GROUP_ID = 0;
+    // Use smaller caches in testing.
```

Comment:
Sorry for nitting again. But can you provide some reasoning in comments (such as using the defaults ends up tests needing much more memory). Just wanted to make sure that the time and effort you spent in order to figure out the problem is preventing someone new not doing the same thing.